ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -103,6 +105,20 @@ class TabBarViewDemoController: DemoController {
         updateBadgeNumbers()
     }
 
+    private func homeItem(shouldShowTitle: Bool) -> TabBarItem {
+        if shouldShowTitle {
+            return TabBarItem(title: "Home",
+                              image: UIImage(named: "Home_24")!,
+                              selectedImage: UIImage(named: "Home_Selected_24")!)
+        } else {
```

Comment:
nit: no need for the else here if you're just returning directly in the if clause.